cpp/src/hash/concurrent_unordered_multimap.cuh

```diff
@@ -634,6 +635,7 @@ public:
                typename comparison_type = key_equal>
     __forceinline__
     __host__ __device__ const_iterator find_part(const key_type& the_key,
+						 int& dest_part,
```

Comment:
Usually I find it a lot clearer at the call-site when a function's output parameters are passed by pointer rather than by reference.